Algorithm.CSharp/CfdTimeZonesRegressionAlgorithm.cs

```diff
@@ -95,7 +95,7 @@ public override void OnData(Slice data)
             {"Tracking Error", "0.006"},
             {"Treynor Ratio", "0"},
             {"Total Fees", "$0.00"},
-            {"Estimated Strategy Capacity", "$0"},
+            {"Estimated Strategy Capacity", "$82000000000.00"},
```

Comment:
82 billion, don't think this is accurate enough?

Reply:
Yeah, its the static factor we are using for both forex and cfd
```_forexMinuteVolume = 25000000m;```

Not sure what value is best 🤔

Reply:
Just looked into this a bit, apparently Forex markets makeup almost all trading volume daily explaining why the static forex volume value is so high.

Goofy picture attached below:
![image](https://user-images.githubusercontent.com/41277998/119033457-18446080-b962-11eb-8e79-5951cf55bb61.png)

After looking at some charts for CFD with volumes I think we can use a very different number. In Fact I am seeing ~300k daily not minute like forex. Maybe I could use a unique estimation value for  CFDs like 200 per minute to get a more realistic value. (200 is roughly 300k a day broken into minutes)